swatch-contracts/TEST_PLAN.md

```diff
@@ -1429,3 +1429,115 @@ This section validates capacity calculations across time boundaries. Tests verif
 - **Expected Results**:
   - HYPERVISOR Sockets measurement = 40
+
+## Metered Cores
+**capacity-metrics-TC005 - Decreasing metered Cores**
+- **Description**: Verify that a metered capacity decreases when the cores quantity is decreased
+- **Test Steps**:
+  1. Given: Create a contract with Rosa (rosa) product that has Core:8
+  2. When: Update the Cores to 4 and run Subscriptions sync
+  3. Then: Verify that the cores are updated (decreased) and the capacity reflects the update sku
+- **Expected Results**: 
+  - the Cores will reflect 4 cores 
+
+**capacity-metrics-TC006 - Increasing metered Cores**
+- **Description**: Verify that a metered capacity increases when the core quantity is increased
+- **Test Steps**:
+  1. Given: Create a contract with a Rosa (rosa) product that has Core:4
+  2. When: Update the Cores to 8 and run Subscriptions sync
+  3. Then: Verify that the cores are updated (increased) and the capacity reflects the updated sku
+- **Expected Results**: 
+  - the Cores will reflect 8 cores 
+
+## Non-metered cores
+**capacity-metrics-TC007 - Decreasing Non metered cores**
+- **Description**: Verify that a non-metered capacity decreases when the core quantity is decreased
+- **Test Steps**:
+  1. Given: create a subscription with OpenShift Container Platform (openshift-container-platform) Cores 8
+  2. When: Update the cores to be 4 and run Subscription sync
+  3. Then: Verify that the cores are updated (decreased) and the capacity reflects the updated sku
+- **Expected Results**: 
+  - the Cores will reflect the 4 cores
+
+**capacity-metrics-TC008 - Increasing Non metered cores**
+- **Description**: Verify that a non-metered capacity increases when the core quantity is increased
+- **Test Steps**:
+  1. Given: create a subscription with OpenShift Container Platform (openshift-container-platform) Cores 4
+  2. When: Update the cores to be 8 and run Subscription sync
+  3. Then: Verify that the cores are updated (increased) and the capacity reflects the updated sku
+- **Expected Results**: 
+  - the Cores will reflect the 8 cores
+
+## Non-metered Sockets
+**capacity-metrics-TC009 - Decreasing Non metered sockets**
+- **Description**: Verify that a non-metered capacity decreases when the sockets quantity is decreased
+- **Test Steps**:
+  1. Given: create a subscription with Satellite Server product (satellite-server) sockets 8
+  2. When: Update the sockets to be 4 and run Subscription sync
+  3. Then: Verify that the sockets are updated (decreased) and the capacity reflects the updated sku
+- **Expected Results**: 
+  - the sockets will reflect the 4 cores
+
+**capacity-metrics-TC010 - Increasing Non metered sockets**
+- **Description**: Verify that a RHEL for x86 (rhel-for-x86) capacity increases when the sockets quantity is increased
+- **Test Steps**:
+  1. Given: create a subscription with sku sockets 4
+  2. When: Update the Satellite Server product (satellite-server) sockets to be 8 and run Subscription sync/Reconcile (?)
```

Comment:
do you mean the RHEL for x86 product? or the RHEL for x86 product should be Satellite Server product instead?